jobs/jobs/databricks_utils.py

```diff
@@ -0,0 +1,83 @@
+import dataclasses
+import json
+import logging
+import os
+from datetime import datetime
+from typing import Iterable, List
+
+from databricks.sdk import WorkspaceClient
+
+import jobs.pipeline as pipeline
+from jobs.schemas import Pipeline
+
+logger = logging.getLogger(__name__)
+logger.setLevel("DEBUG")
+
+
+DATABRICKS_SERVICE_SCHEME = os.getenv("DATABRICKS_SERVICE_SCHEME", "http")
+DATABRICKS_SERVICE_HOST = os.getenv("DATABRICKS_SERVICE_HOST")
+DATABRICKS_SERVICE_PORT = os.getenv("DATABRICKS_SERVICE_PORT")
+
+
+def get_client() -> WorkspaceClient:
+    url = f"{DATABRICKS_SERVICE_SCHEME}://{DATABRICKS_SERVICE_HOST}"
+    if DATABRICKS_SERVICE_PORT:
+        url = f"{url}:{DATABRICKS_SERVICE_PORT}"
+    return WorkspaceClient(host=url, token=os.environ["DATABRICKS_KEY"])
+
+
+def get_pipelines() -> Iterable[Pipeline]:
+    client = get_client()
+    pipelines = client.jobs.list()
+    for pipeline_ in pipelines:
+        yield Pipeline(
+            id=pipeline_.job_id,
+            type="databricks",
+            version=1,
+            date=datetime.today(),  # todo: change to created_time
+            meta={},
+            name=f"{pipeline_.settings.name}",
+        )
+
+
+async def run(
+    pipeline_id: str,
+    job_id: int,
+    files: List[pipeline.PipelineFile],
+    current_tenant: str,
+):
+    logger.info(
+        "Running pipeline %s, job_id %s, current_tenant: %s with arguments %s",
+        pipeline_id,
+        files,
+        job_id,
+        current_tenant,
```

Comment:
I think the order is incorrect:
```suggestion
    logger.info(
        "Running pipeline %s, job_id %s, current_tenant: %s with arguments %s",
        pipeline_id,
        job_id,
        current_tenant,
        files,
```